collectors/hra/parser.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from .record import Record
+from datetime import date
+import logging
+import six
+logger = logging.getLogger(__name__)
+
+
+def parse_response(response, endpoint, from_date, to_date, errors, success, conn):
+    for application in response.json():
+        try:
+            data = {}
+            data['last_updated'] = date.today().strftime('%Y-%m-%d')
+            data['api_date_from'] = from_date.strftime('%Y-%m-%d')
+            data['api_date_to'] = to_date.strftime('%Y-%m-%d')
```

Comment:
# Update

@roll 
- Collector starts at where first records is found (I found this to be Decembar 2008, I checked till 1984 and there are no other records)
- This simplifies the logic significantly, no need to keep parts of the endpoint, less checks, there's only one loop.
- Parse script is only parsing functionality, only response is passed to this function.
- On restart the collector will query the database for the latest `updatedDate` and begin from there.